clickhouse_search/test_utils.py

```diff
@@ -6,14 +6,29 @@
     VARIANT3 as HAIL_VARIANT3,
     VARIANT4 as HAIL_VARIANT4,
     PROJECT_2_VARIANT as HAIL_PROJECT_2_VARIANT,
+    GRCH37_VARIANT as HAIL_GRCH37_VARIANT,
 )
 
 VARIANT1 = {**deepcopy(HAIL_VARIANT1), 'key': 1}
 VARIANT2 = {**deepcopy(HAIL_VARIANT2), 'key': 2}
 VARIANT3 = {**deepcopy(HAIL_VARIANT3), 'key': 3}
 VARIANT4 = {**deepcopy(HAIL_VARIANT4), 'key': 4}
 PROJECT_2_VARIANT = {**deepcopy(HAIL_PROJECT_2_VARIANT), 'key': 5}
-for variant in [VARIANT1, VARIANT2, VARIANT3, VARIANT4, PROJECT_2_VARIANT]:
+GRCH37_VARIANT = {
+    **deepcopy(HAIL_GRCH37_VARIANT),
+    'key': 11,
+    'liftedOverGenomeVersion': '38',
+    'liftedOverChrom': '7',
+    'liftedOverPos': 143271368,
+}
+for genotype in GRCH37_VARIANT['genotypes'].values():
+    genotype['sampleType'] = 'WES'
+GRCH37_VARIANT['predictions'].update({'fathmm': None, 'mut_pred': None, 'vest': None})
```

Comment:
@bpblanken or hail backend fixture data for 37 does not have fathmm, mut_pred or vest (they are 38 only). Just confirming that we do in fact have data for these in 37 in clickhouse